readme.txt

```diff
@@ -142,6 +142,10 @@ Creating or changing routes should always occur in the context of the `wp_router
 
 == Changelog ==
 
+= TBD =
+
+* Fix - Added an additional check for `set_post_contents` so that it only runs on posts that are created via WP-Router.
```

Comment:
Would it be worth it to start including the ticket ID in the readme entries, maybe it could help to track things down in the future.